### ISSUE TITLE
[BUG] Using on server throws error

### ISSUE THREAD

**Issue by ChristianANoeddeboe** [ApacheTech-VintageStory-Mods/Knapster#7]
**Describe the Bug:**
Installing the mod on a server will throw the following on server startup:

```
30.1.2024 18:35:41 [Error] [knapster] Exception: Object reference not set to an instance of an object.
   at Gantry.Services.FileSystem.FileSystemService..ctor(FileSystemServiceOptions options)
   at Gantry.Services.FileSystem.DependencyInjection.GantryDependencyInjectionExtensions.AddFileSystemService(IServiceCollection services, Action1 options)
   at ApacheTech.VintageMods.Knapster.Program.ConfigureUniversalModServices(IServiceCollection services, ICoreAPI api)
   at Gantry.Core.DependencyInjection.ModHost.<>c__DisplayClass6_0.<BuildServerHost>b__1(IServiceCollection ioc)
   at ApacheTech.Common.Extensions.System.ObjectExtensions.With[T](T item, Action1 work)
   at Gantry.Core.DependencyInjection.ModHost.BuildServerHost(ICoreServerAPI sapi)
   at Gantry.Core.DependencyInjection.ModHost.StartPreUniversal(ICoreAPI api)
   at Gantry.Core.ModSystems.Generic.ModSystemBase.StartPre(ICoreAPI api)
   at Vintagestory.Common.ModLoader.TryRunModPhase(Mod mod, ModSystem system, ICoreAPI api, ModRunPhase phase) in VintagestoryLib\Common\API\ModLoader.cs:line 620
```

I copied this from the error log. The server does start, though without the mod activated.

**Comment by ApacheTech** [comment]
I cannot re-produce this issue within v2.7.0, on game v1.19.3.

It's possible that you may have some left over assets from previous versions. Ensure you do not have any previous versions within your mods folder, you mod cache folder, and your mod data folder. That would be my first point of call.

**Comment by ApacheTech** [comment]
This has been addressed, and fixed within v2.9.0.
